kustomize/base/deployment.yaml

```diff
@@ -9,26 +9,19 @@ spec:
   replicas: 1
   revisionHistoryLimit: 1
   selector:
-    app: ${BUGZOOKA_NAMESPACE}
-    deploymentconfig: bugzooka
+    matchLabels:
+      app: ${BUGZOOKA_NAMESPACE}
+      deployment: bugzooka
   strategy:
-    type: Rolling
```

Comment:
How are we making sure auto updates from now onwards? with the new changes coming in.